### PR TITLE
Update get started and authentication page based on backend_configurable value in config

### PR DESCRIPTION
**Pull request by bandinib-amzn**
*Issue #, if available:*

*Description of changes:*

- Updates Get Started intro paragraph text “three” to “these”
- Removes the graphic and `Add backend` step from Get started page if `backend_configurable: false` in config
- Updates the graphics in Get started page to replace external identities text to backend roles.
- Removes config.yml button from `Authentication & Authorization` page if `backend_configurable: false` in config
- Updates UT and snapshots

*Testing:*
```
opendistro_security % yarn test:jest_ui -u --coverage --silent
Test Suites: 55 passed, 55 total
Tests:       300 passed, 300 total
Snapshots:   43 passed, 43 total
Time:        49.708s
✨  Done in 58.82s.
```

*Screenshots:*
when `opendistro_security.ui.backend_configurable: false` in `kibana.yml`

*Get Started page:*
image

image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: public/apps/configuration/panels/get-started.tsx [hunk 1]
```diff
@@ -35,47 +35,48 @@ import { Action, ResourceType } from '../types';
...
+const addBackendStep = {
+  title: 'Add backends',
+  children: (
+    <>
+      <EuiText size="s" color="subdued">
+        Add authentication<EuiCode>(authc)</EuiCode>and authorization<EuiCode>(authz)</EuiCode>
+        information to<EuiCode>plugins/opendistro_security/securityconfig/config.yml</EuiCode>. The
+        <EuiCode>authc</EuiCode> section contains the backends to check user credentials against.
+        The <EuiCode>authz</EuiCode>
+        section contains any backends to fetch external identities from. The most common example of
```
Comment (tianleh) [inline public/apps/configuration/panels/get-started.tsx:47]: Just confirm: is this "external identities" supposed to be replaced?

File: public/apps/configuration/panels/get-started.tsx [hunk 2]
```diff
@@ -35,47 +35,48 @@ import { Action, ResourceType } from '../types';
...
+        an external identity is an LDAP group.{' '}
```
Comment (tianleh) [inline public/apps/configuration/panels/get-started.tsx:48]: same here

File: public/apps/configuration/panels/get-started.tsx (hunk 1 above)
Comment (yuxitang-amzn) [reply public/apps/configuration/panels/get-started.tsx:47]: I think my PR https://github.com/opendistro-for-elasticsearch/security-kibana-plugin/pull/619 has already done the replacement. Rebase to latest and solve conflict should work.

Comment (bandinib-amzn) [reply public/apps/configuration/panels/get-started.tsx]: Rebased to latest and resolved conflict. Now you should see backend roles.

File: public/apps/configuration/panels/get-started.tsx (hunk 2 above)
Comment (bandinib-amzn) [reply public/apps/configuration/panels/get-started.tsx]: updated

**Review by tianleh — APPROVED** [review]

**Review by yuxitang-amzn — APPROVED** [review]

**Pull request merged**